### ISSUE TITLE
could not able to load .vtm file in vtk.js

### ISSUE THREAD

**Issue by anandadake** [Kitware/vtk-js#2588]
### Environment
- **vtk.js**: 
- **OS**: Ubuntu 20.04
- **Browser**: Chrome 106.0.5249.61

**Comment by finetjul** [comment]
Indeed, VTK.js does not support multi-block files. A VTM reader could be written and flattens the VTM file into a vtkPolyData.

**Comment by anandadake** [comment]
Thank you so much for your replay,

is there any demo or tutorial for the same?

**Comment by finetjul** [comment]
> is there any demo or tutorial for the same?

A demo/tutorial to do what ? a VTM reader ? The best is to look at the [VTP reader](https://github.com/Kitware/vtk-js/tree/master/Sources/IO/XML/XMLPolyDataReader)

**Comment by anandadake** [comment]
A demo/tutorial to do  VTM file into vtkPolyData conversion
